app/src/androidTest/java/sdp/moneyrun/LoginInstrumentedTest.java

```diff
@@ -142,12 +147,16 @@ public void logOutWorks(){
             Thread.sleep(1000);
             intended(hasComponent(MenuActivity.class.getName()));
             assertNotNull(FirebaseAuth.getInstance().getCurrentUser());
+            onView(withId(R.id.drawer_layout))
+                    .check(matches(isClosed(Gravity.LEFT)))
+                    .perform(DrawerActions.open());
             Espresso.onView(withId(R.id.log_out_button)).perform(ViewActions.click());
             Thread.sleep(1000);
             assertEquals(FirebaseAuth.getInstance().getCurrentUser(), null);
             Intents.release();
         } catch (InterruptedException e) {
             e.printStackTrace();
+            Intents.release();
```

Comment:
Just a piece of advice, I think it's considered better to do a Intents.release() in a finally clause. I don't think it's needed to change it here but just in case for future tests

Reply:
Yeah, I did not actually write those tests, I just added that line because if one test would fail without releasing the intents, all the other tests relying on intents would fail.

Reply:
Ok no problem